config.yaml

```diff
@@ -39,9 +39,12 @@ options:
     default: 15
   retention-period:
     description: |
-      Sets a global retention period for log streams in Loki. A value of 0 disables retention (default).
-      Minimum retention period is 1d.
-      Specify the period in days. For example, to set a 48-day retention period, use `48`.
+      Sets a global retention period for log streams in Loki.
+      A value of 0 disables retention (default), logs will live forever if no duration is chosen.
```

Comment:
Is this true that they live forever? In a previous PR it was mentioned that `744h (30days)` is used. Just double checking.